openlibrary/templates/merge_queue/comment.html

```diff
@@ -0,0 +1,28 @@
+$def with(comment=None, comment_str=None)
```

Comment:
Add top-level comments describing the difference between `comment` and `comment_str`.